timvt/utils/dependencies.py

```diff
@@ -37,5 +38,23 @@ async def TileMatrixSetParams(
     return morecantile.tms.get(TileMatrixSetId.value)
 
 
+async def TableParams(
+    request: Request, table: str = Path(..., description="Table Name"),
```

Comment:
Do you know how we could get the table argument to be an enum? We get validation from the lookup, but it would be nice to have the available layers show up in the open API docs.

Reply:
so it's possible to do it but I'm starting to be a bit worry that we expose all tables by default.

Reply:
Well this was maybe a lie, right now I don't see an easy way of doing this. I'll keep digging